### DISCUSSION TITLE
Windows下可能已经不太推荐yogadns

### DISCUSSION THREAD

**Discussion by wsvg14** [IrineSistiana/mosdns#203] [General]
yogadns驱动层面代理dns确实方便，但是网络发生变化时容易出问题。
正常运行时配合mosdns的情况下容易报包大小溢出的问题，正常解析dns时也不能立即出结果。
并且在1.22以后无ui界面的服务模式需要付费才能使用，对于windows可能需要退回使用传统修改适配器的方法
其实win11可以整点本地doh(

**Comment by IrineSistiana** [comment]
yogadns 是写给非专业用户看的。传统win修改适配器的方法太反人类了，不适合新手。

~win11还没用上~
